### PR TITLE
Add parameter ConfigFile for Invoke-NugetRestore

### PR DESCRIPTION
**Pull request by shalomich**

### CODE REVIEW

File: modules/Saritasa.Build/Saritasa.Build.psm1 [hunk 1]
```diff
@@ -52,6 +52,9 @@ function Invoke-NugetRestore
         # Path to solution. All NuGet packages from included projects will be restored.
         [Parameter(Mandatory = $true, ParameterSetName = 'Solution')]
         [string] $SolutionPath,
+	  # Path to nuget config file.
```
Comment (AntonZiminSaritasa) [inline modules/Saritasa.Build/Saritasa.Build.psm1:55]: Please use spaces.

File: modules/Saritasa.Build/Saritasa.Build.psm1 [hunk 2]
```diff
@@ -80,6 +83,11 @@ function Invoke-NugetRestore
     if ($SolutionPath)
     {
         $params += $SolutionPath
+
+        if ($ConfigFile)
+        {
```
Comment (AntonZiminSaritasa) [inline modules/Saritasa.Build/Saritasa.Build.psm1:88]: Follow the same style.

File: modules/Saritasa.Build/Saritasa.Build.psm1 [hunk 3]
```diff
@@ -80,6 +83,11 @@ function Invoke-NugetRestore
...
+          $params += @('-ConfigFile', $ConfigFile)
```
Comment (AntonZiminSaritasa) [inline modules/Saritasa.Build/Saritasa.Build.psm1:89]: It should be outside of the `if ($SolutionPath)` to be applied both to solutions and projects.

**Review by AntonZiminSaritasa — CHANGES_REQUESTED** [review]

**Comment by dermeister0** [comment]
#137

**Pull request closed without merging**